lib/screens/transfer/send/send_confirmation/interactor/send_confirmation_bloc.dart

```diff
@@ -16,17 +14,15 @@ class SendConfirmationBloc extends Bloc<SendConfirmationEvent, SendConfirmationS
   Stream<SendConfirmationState> mapEventToState(SendConfirmationEvent event) async* {
     if (event is InitSendConfirmationWithArguments) {
       yield state.copyWith(
-          pageState: PageState.success,
-          lineItems:
-              state.data.entries.map((e) => SendInfoLineItems(label: e.key.inCaps, text: e.value.toString())).toList(),
-          actionName: state.actionName,
-          account: state.account,
-          data: state.data);
+        pageState: PageState.success,
+        transaction: state.transaction,
+      );
```

Comment:
doens't seems this event be required anymore I mean this just copy the same state not sure if the only thing change here is the PageState from initial to success for no reason.

Reply:
True this code seems weird - but it was like this before - I guess it used to set the line items, but we don't do that anymore. 

Would we just remove the whole InitSendConfirmationWithArguments event?